src/main/realtime.ts

```diff
@@ -12,16 +15,22 @@ function cleanAndStartRemoteNotifications() {
   stopRemoteNotifications();
 
   socket = io(process.env.NOTIFICATIONS_URL, {
+    transports: ['websocket', 'polling'],
```

Comment:
If we are not using polling, remove it from the allowed methods. If not, it will fallback to polling, which we do know in advance is not going to work